server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorker.java

```diff
@@ -28,5 +28,5 @@
  * {@link Callable#call()} returns a Boolean which is {@code true} when some a {@link CeTask} was processed,
  * {@code false} otherwise.
  */
-public interface CeWorkerCallable extends Callable<Boolean> {
+public interface CeWorker extends Callable<Boolean> {
```

Comment:
this renaming does not bring any value to the commit
if really necessary, it should be a dedicated commit